bundle-workflow/README.md

```diff
@@ -10,11 +11,34 @@
     - [Performance Tests](#performance-tests)
   - [Sanity Check the Bundle](#sanity-check-the-bundle)
   - [Auto-Generate Manifests](#auto-generate-manifests)
+  - [Component Onboarding](#component-onboarding)
 
 ## OpenSearch Bundle Workflow
 
 The bundle workflow builds a complete OpenSearch distribution from source. You can currently build 1.0, 1.1 and 1.1-SNAPSHOT.
 
+### How it works
+
+This system performs a top-down build of all components required for a specific OpenSearch bundle release. 
+The input to the system is a manifest that defines the order in which components should be built. 
+All manifests for our current releases are [here](/manifests).
+ 
+To build components we rely on a common entry-point in the form of a build.sh script. See [Custom Build Scripts](#custom-build-scripts).
+Within each build script components have the option to place artifacts in a set of directories to be picked up and published on their behalf.  These are
+
+| name               | description                                                                           |
+|--------------------|---------------------------------------------------------------------------------------|
+| /maven   | Include any publications that should be pushed to maven                                         |
+| /plugins | Where a plugin zip should be placed. If included it will be installed during bundle assembly.   |
+| /libs    | Where any additional libs should be placed that are required during bundle assembly             |
```

Comment:
Ths is incomplete, isn't it?

Reply:
outdated now because I let this sit too long, updated it.